### PR TITLE
OSDOCS-5350 Security Profiles Operator 0.5.2 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-5350

Link to docs preview (VPN required):
[Security Profiles Operator 0.5.2](http://file.rdu.redhat.com/antaylor/OSDOCS-5350/security/security_profiles_operator/spo-release-notes.html#spo-release-notes-0-5-2)

Additional information:
This update addresses an underlying CVE, no bug fix-specific information required.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@xiaojiey @rhmdnd  please review when you get a chance, thanks!

**Review by jhrozek — APPROVED** [review]
/lgtm
thank you!

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,21 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-5-2"]
+== Security Profiles Operator 0.5.2
+
+The following advisory is available for Security Profiles Operator 0.5.2:
```
Comment (tmalove) [inline security/security_profiles_operator/spo-release-notes.adoc:18]:
```suggestion
The following advisory is available for the Security Profiles Operator 0.5.2:
```

Comment (tmalove) [reply security/security_profiles_operator/spo-release-notes.adoc:18]: See the IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by sheriff-rh** [comment]
Shipped live. Merging.

**Pull request merged**